kit/assembly.md

```diff
@@ -32,15 +32,15 @@ The following table summarises the connections which need to be made for each bo
 
 Board                                   | Power
 ----------------------------------------|------------------------------
-[Brain Board](/docs/kit/brain_board)    | 5V, using the provided cable
+[Brain Board](/docs/kit/brain_board)    | 12V, must be connected to L2
 [Power Board](/docs/kit/power_board)    | 12V, via the yellow XT60 to the [battery](/docs/kit/batteries)
 [Motor Board](/docs/kit/motor_board)    | 12V
 [Ruggeduino](/docs/kit/ruggeduino)      | via USB
 [Servo Board](/docs/kit/servo_board)    | 12V
 
-In order to connect the Motor and Servo Boards to the Power Board, you will need to create some power cables.
+In order to connect the Brain, Motor and Servo Boards to the Power Board, you will need to create some power cables.
 This should be done using the provided red and black _power wire_ (this is the thicker wire provided in the kit)
- and a pair of green CamCon connectors. The green power wire should only be used to connect motors to the motor board. 
+ and a pair of green CamCon connectors. The green power wire should only be used to connect motors to the motor board.
```

Comment:
I belive the power wire is blue this year

Reply:
Yeah, I think it's actually been blue for a few years now.